src/components/components/ebay-tooltip-overlay/template.marko

```diff
@@ -0,0 +1,34 @@
+<div
+    class=`${data.type}__overlay`
```

Comment:
Since we are not transpiring the Marko templates with babel this template literal will cause issues in older browsers if teams do not have babel configured to run against Marko files (only in Marko 3 and older versions of Marko 4).

Here I think we should use the Marko specific syntax `class="${data.type}__overlay"` instead until we upgrade to Marko 4.